config.py

```diff
@@ -8,19 +8,6 @@
 SWAGGER_UI_DOC_EXPANSION = 'none'
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
+API_TITLE = 'Model Asset Exchange Microservice'
```

Comment:
Didn't we decide to change this value to the name of the model? Or are we doing that in a different PR?